src/components/TaskLine.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Check from './Check';
+import DeleteButton from './DeleteButton';
+import TaskLineCss from '../styles/TaskLine.module.scss';
+
+export default function TaskLine({ item, style }) {
+    const [showDeleteButton, setShowDeleteButton] = React.useState(false);
+    return (
+        <div
+            className={TaskLineCss.taskLine}
+            onMouseEnter={() => setShowDeleteButton(true)}
```

Comment:
This logic can be implemented with pure css

```
.deleteButton  {
 display: none;
}

.taskLine:hover .deleteButton {
 display: block;
}
```

Reply:
done